### PR TITLE
Update focus-lock version

### PR DESCRIPTION
**Pull request by abhimanyu-singh-uber**
Fixes https://github.com/theKashey/react-focus-lock/issues/162

@theKashey We would also need to bump the focus-lock version to fix the above issue.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,5 +10,5 @@ jobs:
       - checkout
       - run: yarn
       - run: yarn lint
-      - run: yarn size
+      - run: NODE_OPTIONS=--openssl-legacy-provider yarn size
```
Comment (abhimanyu-singh-uber) [inline .circleci/config.yml:13]: Added this to fix the following issue.
image
More Details here - https://github.com/webpack/webpack/issues/14532

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "build:es5": "NODE_ENV=es2015 babel src -d dist/es2015",
     "build": "rm -Rf ./dist && yarn build:es5 && yarn build:cjs",
     "test": "npm run test:pick -- '_tests/**/*spec.js'",
-    "test:pick": "NODE_ENV=cjs mocha --require @babel/register --require jsdom-global/register --require _tests/spinup/scaffolding ",
+    "test:pick": "NODE_ENV=cjs mocha --require @babel/register --require jsdom-global/register --require _tests/spinup/scaffolding --exit",
```
Comment (abhimanyu-singh-uber) [inline package.json:17]: this fixes the following issue with yarn test just being stuck and timing out after completing the tests.
image

Comment (theKashey) [reply package.json:17]: OH, I was suffering from this for quite a while and was going to replace mocha by (a more common for me) jest next week.
Thank you!

**Comment by theKashey** [comment]
🤝

**Review by theKashey — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (theKashey) [reply .circleci/config.yml:13]: Thanks. I was close to switch to old good Travis.

**Comment by abhimanyu-singh-uber** [comment]
Thanks, what do we do about `2.7.0` version? let it stay on npm ?

**Comment by abhimanyu-singh-uber** [comment]
Please ping me here once a new version is released with this change. Thanks

**Comment by theKashey** [comment]
2.7.1 has been just released.
2.7.0 will be deprecated soon after

**Comment by abhimanyu-singh-uber** [comment]
Yay! party!

**Comment by abhimanyu-singh-uber** [comment]
also, confirmed the prevent scroll is working with this release. Thanks

**Pull request merged**